### DISCUSSION TITLE
Metrics for elastic batches

### DISCUSSION THREAD

**Discussion by timfehr** [vectordotdev/vector#21703] [Q&A] [answered]
Hi all, is there a way the get some metrics of how big the batches are that are send by the elastic sink to an endpoint. Are there any metrics or logs present. And maybe also which of the three settings for batches triggered the batch. That would be really helpful for debugging and optimizing the batches.

**Comment by pront** [comment, marked as answer]
Hi @timfehr, currently there's no instrumentation for this. 

See:
* https://github.com/vectordotdev/vector/issues/9719
* https://github.com/vectordotdev/vector/issues/20284

I agree this is super useful to have.
